### ISSUE TITLE
Can version 1.6 also use MySQL database?

### ISSUE THREAD

**Issue by Iverson-lena** [goharbor/harbor#5653]
The 1.6 version is as follows(Migrate database to Postgresql).
Can I continue to use MySQL database docking harbor? there is no configuration method can not migrate to the postgre database, thank you.

**Comment by reasonerjt** [comment]
No, you have to follow the migration guide to migration to pgSQL.
In 1.6 an important update is to merge into one DB instance, which is crucial for future HA deployment support.
